### ISSUE TITLE
is anyone actively using this? I have a question

### ISSUE THREAD

**Issue by roastbeeef** [fopina/pyspeedtest#18]
I need to know the what unit of measurement the Python output (not CLI) is measured in.  I'm trying to match this up to my connection speed but it's not very close.  A bit of assistance from someone actively using this would be great.

thanks,
matt

**Comment by fopina** [comment]
Hey @roastbeeef indeed I've been delaying what I want to do with this module which is re-do it all over (too long since I first released it) so I haven't been properly maintaining it..

One of the things missing is indeed documenting it!

But code is always the most accurate documentation and in this case, the only one :)

https://github.com/fopina/pyspeedtest/blob/master/pyspeedtest.py#L359

https://github.com/fopina/pyspeedtest/blob/master/pyspeedtest.py#L401

So basically, unit is bytes per second
